MonoGame.Framework/Platform/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -317,11 +341,19 @@ internal void CreateSizeDependentResources()
             // If the swap chain already exists... update it.
             if (_swapChain != null)
             {
+                var swapChainFlags = SwapChainFlags.None;
+#if WINDOWS_UAP
+                if (PresentationParameters.PresentationInterval == PresentInterval.Immediate)
+                {
+                    swapChainFlags = SwapChainFlags.AllowTearing;
+                }    
+#endif
```

Comment:
If you move the code here out of the if statement you can use the  swapChainFlags variable below in the initializer when creating a new swapchain and remove the duplicate code.

Reply:
Nice catch, I didn't notice. Will improve this.